pages/about/groups/ig.md

```diff
@@ -50,80 +50,53 @@ The group provides frequent opportunities for you to give input on digital acces
 
 ## WAI Interest Group mailing lists {#mailinglist}
 
-The WAI Interest Group has two mailing lists.
+The WAI Interest Group has two mailing lists with public archives:
+1. **Announcements list** (public-wai-announce)  &mdash; for WAI announcements only. [Announcements archives](http://lists.w3.org/Archives/Public/public-wai-announce/)
+2. **Discussion list** (w3c-wai-ig) &mdash; for public discussion and WAI announcements. [Discussion archives](http://lists.w3.org/Archives/Public/w3c-wai-ig/)
```

Comment:
@tamsinewing555 OK to unbold **Announcements list** and **Discussion list** if you think better not.